pkg/networkservice/core/trace/client.go

```diff
@@ -50,8 +64,6 @@ func (t *traceClient) Request(ctx context.Context, request *networkservice.Netwo
 	ctx, finish := withLog(ctx, operation)
 	defer finish()
 
-	logRequest(ctx, request)
```

Comment:
You still need to remember current request before it will be modified with traced.
```connInfo.Request = proto.Clone(request)```